src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/DecisionGroupManager.java

```diff
@@ -0,0 +1,217 @@
+package de.uhd.ifi.se.decision.management.jira.persistence;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.atlassian.activeobjects.external.ActiveObjects;
+
+import de.uhd.ifi.se.decision.management.jira.ComponentGetter;
+import de.uhd.ifi.se.decision.management.jira.model.DocumentationLocation;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeElement;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeGraph;
+import de.uhd.ifi.se.decision.management.jira.persistence.impl.JiraIssuePersistenceManager;
+import de.uhd.ifi.se.decision.management.jira.persistence.tables.DecisionGroupInDatabase;
+import net.java.ao.Query;
+
+/**
+ * Class responsible for groups of decision from all Jira based documentation
+ * locations. Groups are stored in the internal database of Jira.
+ * 
+ * @see KnowledgePersistenceManager
+ * @see DecisionGroupInDatabase
+ * @see JiraIssuePersistenceManager
+ */
+public class DecisionGroupManager {
+
+    public static final ActiveObjects ACTIVE_OBJECTS = ComponentGetter.getActiveObjects();
+
+    public static boolean deleteGroupAssignment(Long elementId) {
+	if (elementId == null) {
+	    return false;
+	}
+	boolean isDeleted = false;
+	for (DecisionGroupInDatabase dgData : ACTIVE_OBJECTS.find(DecisionGroupInDatabase.class)) {
+	    if (dgData.getId() == elementId) {
+		isDeleted = DecisionGroupInDatabase.deleteGroup(dgData);
+	    }
+	}
+	return isDeleted;
+    }
+
+    public static boolean deleteGroupAssignment(String group, KnowledgeElement element) {
+	if (element == null || group == null) {
+	    return false;
+	}
+	boolean isDeleted = false;
+	for (DecisionGroupInDatabase dgData : ACTIVE_OBJECTS.find(DecisionGroupInDatabase.class)) {
+	    if (dgData.getGroup() == group && dgData.getSourceId() == element.getId()) {
+		isDeleted = DecisionGroupInDatabase.deleteGroup(dgData);
+	    }
+	}
+	return isDeleted;
+    }
+
+    /**
+     * Replaces all current existing assignments for that element with a list of new
+     * groups
+     * 
+     * @param List of groups to add
+     * @param The  element that the groups should be assigned to
+     * @return If replacement was successful
+     */
+    public static boolean setGroupAssignment(List<String> groups, KnowledgeElement element) {
+	if (groups == null || element == null) {
+	    return false;
+	}
+	List<String> oldGroups = getGroupsForElement(element);
+	boolean success = false;
+	long id = 0;
+	for (String oldGroup : oldGroups) {
+	    success = deleteGroupAssignment(oldGroup, element);
+	}
+	for (String group : groups) {
+	    id = insertGroup(group, element);
+	}
+	if (success == true && id != -1) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary comparisons in boolean expressions](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5160185)